user-management-service/app/src/main/java/com/microservices/usermanagement/entity/UserEntity.java

```diff
@@ -0,0 +1,22 @@
+package com.microservices.usermanagement.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@Table(name = "users")
+public class UserEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    @Column
```

Comment:
1. add blank lines separating fields.
2. remove @Column annotations where it is not nesessary

Reply:
fixed